packages/pages/src/common/src/parsers/sourceFileParser.ts

```diff
@@ -150,21 +158,25 @@ export default class SourceFileParser {
       .fixMissingImports()
       .organizeImports()
       .fixUnusedIdentifiers();
-    this.sourceFile.addStatements('import * as React from "react";');
   }
 
   getFileName(): string {
     return this.sourceFile.getBaseName();
   }
 
-  clearAll() {
-    this.sourceFile.removeText(
-      this.sourceFile.getPos(),
-      this.sourceFile.getEnd()
-    );
-  }
-
   async save() {
     await this.sourceFile.save();
   }
+
+  private getAbsolutePath = (base: string, relative: string): string => {
```

Comment:
Can you add tsdoc to this explaining what it does with an example? Also add tests for it. If it's specific to imports you might want to name it accordingly as well.

Reply:
Also, any reason this is a private function in the class and not just a const function outside of it (though in the same file)?